lib/openstudio-standards/standards/necb/common/btap_data.rb

```diff
@@ -409,7 +409,7 @@ def utility(model)
     building_type = 'Commercial'
     province = provinces_names_map[model.getWeatherFile.stateProvinceRegion]
     neb_eplus_fuel_map = { 'Electricity' => 'Electricity',
-                           'Natural Gas' => 'Gas',
+                           'Natural Gas' => 'Natural Gas',
```

Comment:
This is a map from the NEB names to the EnergyPlus naming style.. So this line should be...
`'Natural Gas' => 'NaturalGas'`